core/src/main/scala/cats/syntax/either.scala

```diff
@@ -153,6 +153,12 @@ final class EitherOps[A, B](val eab: Either[A, B]) extends AnyVal {
     case r @ Right(_) => EitherUtil.leftCast(r)
   }
 
+  def flatTap[AA >: A, D](f: B => Either[AA, D]): Either[AA, B] =
```

Comment:
Here too.

Reply:
This version allows widening of A. `Monad`'s one doesn't.